### ISSUE TITLE
Rstudio Freeze when viewing output object

### ISSUE THREAD

**Issue by ivlachos** [shosaco/vistime#5]
I ran the presidents example for a quick test of the package and RStudio viewer froze with a white background.

The code is copy pasted from the website and/or ?vistime.

```
pres <- data.frame(Position = rep(c("President", "Vice"), each = 3),
                  Name = c("Washington", rep(c("Adams", "Jefferson"), 2), "Burr"),
                  start = c("1789-03-29", "1797-02-03", "1801-02-03"),
                  end = c("1797-02-03", "1801-02-03", "1809-02-03"),
                  color = c('#cbb69d', '#603913', '#c69c6e'),
                  fontcolor = c("black", "white", "black"))
                  
vistime(pres, events="Position", groups="Name", title="Presidents of the USA", lineInterval = 60*60*24*365*5)

```

My session info is as follows:

```
R version 3.4.3 (2017-11-30)
Platform: x86_64-redhat-linux-gnu (64-bit)
Running under: CentOS Linux 7 (Core)

Matrix products: default
BLAS/LAPACK: /usr/lib64/R/lib/libRblas.so

locale:
[1] LC_CTYPE=en_US.UTF-8 LC_NUMERIC=C LC_TIME=en_US.UTF-8 LC_COLLATE=en_US.UTF-8 LC_MONETARY=en_US.UTF-8 LC_MESSAGES=en_US.UTF-8 LC_PAPER=en_US.UTF-8 LC_NAME=C LC_ADDRESS=C
[10] LC_TELEPHONE=C LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C

attached base packages:
[1] stats graphics grDevices utils datasets methods base

other attached packages:
[1] vistime_0.5.0 plotly_4.8.0 ggplot2_3.0.0 data.table_1.10.4-3

loaded via a namespace (and not attached):
[1] Rcpp_0.12.17 bindr_0.1.1 magrittr_1.5 tidyselect_0.2.4 munsell_0.4.3 viridisLite_0.3.0 colorspace_1.3-2 R6_2.2.2 rlang_0.2.1 httr_1.3.1 plyr_1.8.4 dplyr_0.7.6 tools_3.4.3
[14] grid_3.4.3 gtable_0.2.0 withr_2.1.2 htmltools_0.3.6 yaml_2.1.16 lazyeval_0.2.1 assertthat_0.2.0 digest_0.6.15 tibble_1.4.2 bindrcpp_0.2.2 tidyr_0.8.0 purrr_0.2.5 htmlwidgets_1.2
[27] glue_1.2.0 compiler_3.4.3 pillar_1.2.3 scales_0.5.0 jsonlite_1.5 pkgconfig_2.0.1
```

Thank you

**Comment by et260292** [comment]
Got the same problem!  Did you figure out how to fix it?

**Comment by jodjo86** [comment]
Got the same problem :(

**Comment by ivlachos** [comment]
Haven't found a way to fix it. Ended up using timevis instead.

**Comment by shosaco** [comment]
This was due to the new plotly package 4.8.0. Fixed in vistime 0.6.0

**Comment by shosaco** [comment]
looping you in @ivlachos @jodjo86 @et260292 @
